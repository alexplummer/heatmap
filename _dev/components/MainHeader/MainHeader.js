
// MainHeader
// ============
// Heading for the document

// Imports
import React from 'react';

class MainHeader extends React.Component {

    constructor(props) {
        super(props);
    }

    render() {
        return (
            <div className="MainHeader">
                <p className="test">MainHeader loaded!</p>
            </div>
        );
    }
}

// Exports
export default MainHeader;
