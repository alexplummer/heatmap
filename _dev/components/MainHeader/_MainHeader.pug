
//- MainHeader
//- ============
//- Heading for the document

//- Includes

mixin MainHeader()
    
    .MainHeader

        h1 Leatherhead Social Heatmap

        p Click these places to check-in:

        ul
            li.icon-beer Pubs
            li.icon-wine Bars
            li.icon-food Restaurants
