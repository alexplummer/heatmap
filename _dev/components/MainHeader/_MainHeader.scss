
// MainHeader
// ============
// Heading for the document

.MainHeader {
    position: absolute;
    z-index: $z-m;
    background: fade-out($clr-primary, .2);
    border-bottom-right-radius: 20px;

    h1 {
        padding: 0;
        margin: $size-m;
        font-weight: 300;
        color: #fff;
    }
}