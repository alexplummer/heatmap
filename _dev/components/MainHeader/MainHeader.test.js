
// MainHeader
// ============
// Heading for the document

// Imports
import React from 'react';
import renderer from 'react-test-renderer';
import MainHeader from './MainHeader';

describe('MainHeader renders correctly', () => {
    it('renders correctly', () => {
        const rendered = renderer.create(
            <MainHeader />
        );
        expect(rendered.toJSON()).toMatchSnapshot();
    });
});