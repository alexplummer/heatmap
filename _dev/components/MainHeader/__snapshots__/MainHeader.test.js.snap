// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainHeader renders correctly renders correctly 1`] = `
<div
  className="MainHeader"
>
  <p
    className="test"
  >
    MainHeader loaded!
  </p>
</div>
`;
